File: app/src/main/java/com/artmaster/android/orthodoxcalendar/data/di/modules/ContextModule.java
package com.artmaster.android.orthodoxcalendar.data.di.modules;

import android.content.Context;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class ContextModule {
    private Context context;

    public ContextModule(Context context) {
        this.context = context;
    }

    @Singleton
    @Provides
    public Context context() {
        return context;
    }
}

File: app/src/main/java/com/artmaster/android/orthodoxcalendar/data/di/modules/ContextModule.kt
package com.artmaster.android.orthodoxcalendar.data.di.modules

import android.content.Context
import dagger.Module
import dagger.Provides
import javax.inject.Singleton

@Module
class ContextModule(private val context: Context) {
    @Singleton
    @Provides
    fun context(): Context {
        return context
    }
}